src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -284,9 +278,12 @@ void SetGridIC( real fluid[], const double x, const double y, const double z, co
    fluid[MOMZ] = Momz;
    fluid[ENGY] = Etot;
 #  if ( EOS == EOS_NUCLEAR )
-   if ( CCSN_Prob == Post_Bounce )   fluid[YE] = Ye*Dens;   // electron fraction [dens]
+   fluid[YE  ] = Ye*Dens;   // electron fraction [dens]
```

Comment:
Use the already defined array `Passive[]` to improve consistency. For example, just do `for (int v=0; v<NCOMP_PASSIVE; v++) fluid[ NCOMP_FLUID + v ] = Passive[v];`

Reply:
Updated